src/rgw/rgw_lc.h

```diff
@@ -86,7 +86,7 @@ class LCExpiration
   bool valid() const {
     if (!days.empty() && !date.empty()) {
       return false;
```

Comment:
not sure it's critical, but these consecutive tests have !days.empty() in common--combine them?